src/Configuration/Compiler/TaskCompilerPass.php

```diff
@@ -34,7 +34,7 @@ public function process(ContainerBuilder $container)
             }
 
             $tasksRegistered[] = $configKey;
-            if (!array_key_exists($configKey, $configuration)) {
+            if (!array_key_exists($configKey, $configuration?:[])) {
```

Comment:
I would leave this as it is and change line 22.
`$configuration = $container->getParameter('tasks') ?: [];`